charts/rpe-send-letter-service/values.yaml

```diff
@@ -0,0 +1,31 @@
+java:
+  applicationPort: 8485
+  environment:
+    # db
+    LETTER_TRACKING_DB_HOST: "{{ .Release.Name }}-postgresql"
+    LETTER_TRACKING_DB_NAME: "{{ .Values.postgresql.postgresqlDatabase}}"
+    LETTER_TRACKING_DB_USER_NAME: "{{ .Values.postgresql.postgresqlUsername}}"
+    LETTER_TRACKING_DB_PASSWORD: "{{ .Values.postgresql.postgresqlPassword}}"
+    # db - migration
+    FLYWAY_URL: "jdbc:postgresql://{{ .Release.Name }}-postgresql:5432/{{ .Values.postgresql.postgresqlDatabase}}"
+    FLYWAY_NOOP_STRATEGY: "false"
+    # encryption
+    ENCRYPTION_ENABLED: "false"
+    ENCRYPTION_PUBLIC_KEY: ""
+    # schedules
+    SCHEDULING_ENABLED: "true"
+    # inherited
+    S2S_NAME: "send_letter_tests"
+    S2S_URL: "http://rpe-service-auth-provider-aat.service.core-compute-aat.internal"
+    FTP_HOSTNAME: "cmseft.services.xerox.com"
+    FTP_PORT: "22"
+    FTP_FINGERPRINT: "SHA256:3tX3DIkqd1Loz2alHfnt+qjHocfxk0YUOZHlnf9Zgdk"
+    FTP_TARGET_FOLDER: "TO_XEROX"
+    FTP_REPORTS_FOLDER: "FROM_XEROX"
+    FTP_DOWNTIME_FROM: "23:58"
+    FTP_DOWNTIME_TO: "23:59"
+    # mails
+    SPRING_MAIL_HOST: "false"
+  # Don't modify below here
+  image: ${IMAGE_NAME}
```

Comment:
shouldn't be hardcoded: https://github.com/hmcts/draft-store/blob/master/charts/draft-store-service/values.yaml#L12